src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpCertificateHelper.cs

```diff
@@ -11,8 +11,8 @@ namespace System.Net.Http
 {
     internal static class WinHttpCertificateHelper
     {
-        private const string ClientAuthenticationOID = "1.3.6.1.5.5.7.3.2";
-        private static readonly Oid s_serverAuthOid = new Oid("1.3.6.1.5.5.7.3.1");
+		private const string ClientAuthenticationOID = "1.3.6.1.5.5.7.3.2";
```

Comment:
The PR author used a tab instead of spaces.  It got merged as is.
For this port to release/2.0 branch, I simply did a git cherry-pick.

If you want, I'll push another commit to fix the tab.